frontend_tests/node_tests/narrow.js

```diff
@@ -436,6 +436,33 @@ run_test("show_empty_narrow_message", ({mock_template}) => {
         ),
     );
 
+    people.add_active_user(alice);
+    set_filter([["pm-with", alice.email]]);
+    hide_all_empty_narrow_messages();
+    narrow_banner.show_empty_narrow_message();
+    assert.equal(people.verify_non_active_human_id(alice.user_id), false);
+    assert.equal(
+        $(".empty_feed_notice_main").html(),
+        empty_narrow_html(
+            "translated: You have no private messages with Alice Smith yet.",
+            'translated HTML: Why not <a href="#" class="empty_feed_compose_private">start the conversation</a>?',
+        ),
+    );
```

Comment:
This block of code is not needed I think, since this was checked immediately after enabling sending private messages

Reply:
Some of the node_tests are failing without this piece of code, I have attached the log also this was discussed on [CZO](https://chat.zulip.org/#narrow/stream/49-development-help/topic/.E2.9C.94.20node.20test.20for.20.2323899)

![Screenshot from 2022-12-24 23-46-52](https://user-images.githubusercontent.com/66828942/209447750-33bcf2ad-2bfa-4cc7-b685-03878148694d.png)

Please do suggest, if some more changes are required, Thanks.

Reply:
Did you remove the code both above and below this comment? Sorry if I wasn't clear before, but I meant to only refer to the new code above this comment. In other words, let the new code from `people.deactivate(alice)` stay as is and remove the block right above. The tests should still work then

Reply:
Looks like you're removing the code blocks both above and below this comment. Remove just the one above. I've tested locally too, that works (node tests don't fail)

Reply:
Hey @N-Shar-ma ,Made the changes suggested by you, kindly review it  and suggest if any more changes  are required, Thanks.